engine/src/cython/initialize.cpp

```diff
@@ -754,10 +754,8 @@ std::pair<std::pair<std::shared_ptr<CacheMachine>,std::shared_ptr<CacheMachine>
 		std::map<std::string, comm::node> nodes_info_map;
 
 		comm::blazing_protocol protocol = comm::blazing_protocol::tcp;
-		if(config_options.find("PROTOCOL") != config_options.end()){
-			if(config_options["PROTOCOL"] == "UCX"){
-				protocol = comm::blazing_protocol::ucx;
-			}
+		if (config_options["PROTOCOL"] == "ucx"){
```

Comment:
Could you please make it so that its capitalization agnostic when it checks this. So like apply to_lower() or whatever the c++ version of that is